modules/core/src/webgl/vertex-array.js

```diff
@@ -318,6 +304,30 @@ export default class VertexArray {
     return -1;
   }
 
+  _setAttribute(locationOrName, value) {
+    if (value instanceof Buffer) {
+      //  Signature: {attributeName: Buffer}
+      this.setBuffer(locationOrName, value);
+    } else if (Array.isArray(value) && value.length && value[0] instanceof Buffer) {
+      // Signature: {attributeName: [buffer, accessor]}
+      const buffer = value[0];
+      const accessor = value[1];
+      this.setBuffer(locationOrName, buffer, accessor);
+    } else if (ArrayBuffer.isView(value) || Array.isArray(value)) {
+      // Signature: {attributeName: constant}, constant == short (typed) array
+      const constant = value;
+      this.setConstant(locationOrName, constant);
+    } else if (value.buffer instanceof Buffer) {
+      // luma.gl v7: Support accessor objects with 'buffer' field
+      // for interleaved data
+      // Signature: {attributeName: {...accessor, buffer}}
```

Comment:
👍 I like the signature comments but please make them
new Buffer(gl, ......)

Reply:
These are not parameters to `Buffer` but the values of each entry in the attributes map.

Are you suggesting:

```
Signature: {attributeName: {...accessor, buffer: new Buffer(gl, ...)}}
```

I think the key is updating this in a good way in the docs, not here. Probably more should be done in that regard.